### PR TITLE
feat: Feast/IKV online store contrib plugin integration

### PR DESCRIPTION
**Pull request by pushkarmoi**
**What this PR does / why we need it**:
We add a new online store (IKV - https://www.inlined.io/) to feast by following the guide here:
docs.feast.dev/how-to-guides/customizing-feast/adding-support-for-a-new-online-store

### CODE REVIEW

**Review by franciscojavierarceo — APPROVED** [review]
🚀🚀🚀

File: setup.py [hunk 1]
```diff
@@ -209,6 +213,7 @@
     + CASSANDRA_REQUIRED
     + AZURE_REQUIRED
     + ROCKSET_REQUIRED
+    + IKV_REQUIRED
```
Comment (tokoko) [inline setup.py:216]: This is not necessary, since we are not adding tests to run on ci at this point

Comment (pushkarmoi) [reply setup.py:216]: thanks for flagging, removed

**Comment by pushkarmoi** [comment]
Closed this and opened https://github.com/feast-dev/feast/pull/4068 to work-around DCO issue

**Pull request closed without merging**